src/algebra/quaternion.lean

```diff
@@ -153,9 +153,6 @@ instance : algebra R ℍ[R, c₁, c₂] :=
 @[simp] lemma smul_im_j : (r • a).im_j = r • a.im_j := rfl
 @[simp] lemma smul_im_k : (r • a).im_k = r • a.im_k := rfl
 
-@[simp] lemma smul_mk (re im_i im_j im_k : R) :
-  r • (⟨re, im_i, im_j, im_k⟩ : ℍ[R, c₁, c₂]) = ⟨r • re, r • im_i, r • im_j, r • im_k⟩ := rfl
-
```

Comment:
Something has gone wrong with one of your merges, I recently added this.

Reply:
Ooops.... Hopefully all fixed up now.